index.html

```diff
@@ -184,3 +185,14 @@ <h1 class="hero-title">Build and test software of any size, quickly and reliably
     </div>
   </div>
 </div>
+
+<script>
+  var installBtn = document.getElementById("btn-install");
+  if (navigator.appVersion.indexOf("Win") != -1) {
+    installBtn.href = "{{ site.docs_site_url }}/install-windows.html";
+  } else if (navigator.appVersion.indexOf("Mac") != -1) {
+    installBtn.href = "{{ site.docs_site_url }}/install-os-x.html";
+  } else if (navigator.appVersion.indexOf("Linux") != -1) {
+    installBtn.href = "{{ site.docs_site_url }}/install-ubuntu.html";
+  }
```

Comment:
I would say it's safer to land on the install index: https://docs.bazel.build/versions/master/install.html, which is what it is doing in this version.